requirements.txt

```diff
@@ -1,6 +0,0 @@
-Django>=1.6
-amqpconsumer==1.5
-mock==1.3.0
-nose==1.3.7
-tox==2.3.1
-six==1.11.0
```

Comment:
Why is `six` not required? I check and I thought it wasn't present by default? And I don't see you've removed the `six.iteritems` reference

Reply:
My bad, I actually need to make a separate PR for this, because `six` needs to be in `setup.py` not `requirements.txt`

Reply:
https://github.com/ByteInternet/django-eventhandler/pull/12